spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/offer/ContractOffer.java

```diff
@@ -99,7 +99,7 @@ public ZonedDateTime getContractStart() {
         return contractStart;
     }
 
-    @Nullable
+    @NotNull
     public ZonedDateTime getContractEnd() {
         return contractEnd;
```

Comment:
AFAIK @ndr_brt has already commented it, but at this point in time, I would strongly suggest to remove the `ZonedDateTime` in favor of UTC

Reply:
As mentioned above, `ZonedDateTime` is only here for IDS-mapping purposes. Will get rid of that when new protocol is ready.